ext/tiff/CMakeLists.txt

```diff
@@ -207,7 +207,8 @@ message(STATUS "")
 #[===[ Inviwo modification ]===]
 ivw_suppress_compiler_warnings(tiff)
 ivw_default_install_targets(tiff)
-ivw_folder(tiff ext)
+ivw_folder(tiff ext/tiff)
+ivw_folder(release ext/tiff)
```

Comment:
better to remove the include(Release) higher up instead...

Reply:
Yes and no. The release writes the version in the header. Not sure if any code relies on that

Reply:
Should not be needed there is already a generated file in the in the source tree. So Im fairly sure we will not need this.

Reply:
Also I really dislike having a generic target name like that around if it not inviwo related...